anti-abuse/wordban/wordban.js

```diff
@@ -51,7 +51,7 @@ module.exports = function(core) {
 						var customPhrases = room.params.antiAbuse.customPhrases;
 						if (customPhrases instanceof Array && customPhrases.length !== 0) {
 							filters.custom = new RegExp("\\b" + customPhrases.map(function(e) {
-								return "\\b" + e + "\\b";
```

Comment:
1. Which banned word has star in it? can't we just change it in the banned word list instead of doing this?
2. Wrong slash used for escaping